test/ginkgo/Makefile

```
@@ -51,7 +51,8 @@ endif
 
 .PHONY: deploy-argocd-operator undeploy-argocd-operator kustomize test-e2e \
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)